deployment/ccip/changeset/testhelpers/test_environment.go

```diff
@@ -60,6 +61,7 @@ type TestConfigs struct {
 	NumOfRMNNodes              int
 	LinkPrice                  *big.Int
 	WethPrice                  *big.Int
+	MineBlocks                 bool
```

Comment:
If we're gonna do something like this feels like this should just default to true rather than having to specify it explicitly

Reply:
Actually I feel like if you're gonna remove the backend.Commit() logic from the testhelpers this shouldn't even be an option, it should always be done for memory envs, for docker envs they're on automine and configured separately.

Reply:
I was considering that but the same environment is used in deployment tests where we do use Confirm which internally (that should also make the tests slightly faster since the block is immediately created rather than within a 2s window). The mining is only necessary on code that relies on DoSendRequest since the transactions happen within the node.

So my thought was to enable mining for smoke tests and keep it disabled elsewhere. But I don't have a strong preference, I agree that it might be cleaner to just always enable it and we'll just produce a couple more blocks than needed in some tests

Reply:
I changed it to be named `BlockTime` with a default of `2s`, setting to `0` disables